pkg/apis/seedmanagement/types_gardenlet.go

```diff
@@ -80,3 +80,8 @@ type GardenletStatus struct {
 	// Gardenlet's generation, which is updated on mutation by the API Server.
 	ObservedGeneration int64
 }
+
+const (
+	// GardenletReconciled is a condition type for indicating whether the Gardenlet's has been reconciled.
```

Comment:
Thanks for your PR! LGTM.

Except, maybe grammar could be improved here.

```suggestion
	// GardenletReconciled is a condition type for indicating whether the Gardenlet has been reconciled.
```